app/code/Magento/Checkout/view/frontend/templates/cart/totals.phtml

```diff
@@ -19,4 +19,16 @@
                 }
             }
     </script>
+    <script>
+        window.checkoutConfig = <?= /* @escapeNotVerified */ $block->getSerializedCheckoutConfig() ?>;
+        window.customerData = window.checkoutConfig.customerData;
+        window.isCustomerLoggedIn = window.checkoutConfig.isCustomerLoggedIn;
+        require([
```

Comment:
Please move the javascript implementation to js file according to https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/javascript/js-resources.html . @VladimirZaets can you plase advice on proper javascript placement in this case?

Reply:
@sivaschenko it's done.